README.rst

```diff
@@ -81,20 +81,20 @@ physical constants and commonly used constants.
 
 The typical usage is the following:
 
-.. code-block:: python
+.. code-block:: pycon
```

Comment:
":: pycon"? I'm afraid I never came across that one, funny enough. OK, thanks.

Reply:
pre-commit was swallowing the warning about `>>>` not being valid Python. Switching to pycon (Python Console) caused the warning about the print statement to show up (and the blocks to be formatted).

Reply:
Nice.